vaadin-map-flow-parent/vaadin-map-flow/src/main/java/com/vaadin/flow/component/map/configuration/source/XYZSource.java

```diff
@@ -33,7 +33,10 @@ public String getType() {
         return Constants.OL_SOURCE_XYZ;
     }
 
-    protected static class BaseOptions<T extends BaseOptions<T>>
+    public static abstract class BaseOptions<T extends BaseOptions<T>>
```

Comment:
This really shouldn't be public, but it can not be used in the XYZSource constructor otherwise. Made it abstract instead to indicate that it's not intended for direct use.